CNAME

```diff
@@ -1 +0,0 @@
-www.covidcaremap.org
```

Comment:
This is no longer needed because it is specific to GitHub Pages. After migrating to Netlify, we will have to explicitly disable GitHub Pages in the repository **Settings**.